core/core_constants.cpp

```diff
@@ -651,6 +651,7 @@ void register_global_constants() {
 	BIND_CORE_ENUM_CONSTANT(PROPERTY_USAGE_DEFAULT);
 	BIND_CORE_ENUM_CONSTANT(PROPERTY_USAGE_DEFAULT_INTL);
 	BIND_CORE_ENUM_CONSTANT(PROPERTY_USAGE_NO_EDITOR);
+	BIND_CORE_ENUM_CONSTANT(PROPERTY_USAGE_READ_ONLY);
```

Comment:
This should be bound in the same order it's declared in the enum (as the binding order is kept in the docs).